storage/storage-samples/storage-di-sample/src/main/java/org/fido/iot/storage/DiDbManager.java

```diff
@@ -26,14 +26,14 @@ public void createTables(DataSource dataSource) {
 
       String sql = "CREATE TABLE IF NOT EXISTS "
           + "MT_DEVICES("
-          + "GUID CHAR(36) PRIMARY KEY, "
-          + "SERIAL_NO VARCHAR(255), "
+          + "GUID CHAR(36), "
+          + "SERIAL_NO VARCHAR(255) PRIMARY KEY, "
```

Comment:
Component sample Device sends the serial number as '0', everytime, which makes it non-unique for now. Can we please revert this, and do this when device is updated?

Reply:
If a device reruns DI with the same serial no (as it should) then the entry in this table will be replaced with an updated guid, a new row should not be added if the serial no is the same. Can you explain the use case where you see an issue?

Reply:
This is not really an issue in any real-world use-case, and, agree with you on the functionality aspect there.
Thought that we'd like to keep the old vouchers around for general debug purposes (say to compare vouchers during dev to fix issues, if needed, considering pri device is for learning purposes as well.. but we can do with DI-client-sample as well).
Having said that, serial no as PK is indeed better for real-world use-cases.